app/helpers/home_helper.rb

```diff
@@ -0,0 +1,2 @@
+module HomeHelper
```

Comment:
This can be removed, as it doesn't accomplish much right now :) We can always add it back if needed

Reply:
Check :)